docs/gadget-devel/gadget-wasm-api-raw.md

```diff
@@ -341,3 +341,51 @@ Parameters:
 
 Return value:
 - 0 in case of success, 1 otherwise.
+
+### eBPF Maps
+
+#### `getMap(name string) uint32`
+
+Get a handle to an existing eBPF map.
+
+Parameters:
+- `name` (string): Map's name
+
+Return value:
+- (u32) Handle to the data source on success, 0 on error.
+
+#### `mapLookup(m uint32, keyptr uint64, valueptr uint64) uint32`
+
+Lookup the map for a value corresponding to given key.
+
+Parameters:
+- `m` (u32): Map handle (as returned by `getMap`)
+- `keyptr` (u64): A `bufPtr` to data corresponding to key.
```

Comment:
Should we describe what a bufPtr is on "Data types"?

Reply:
The bufPtr concept is no described on this file before. I think we need to describe a bufPtr is a uint64 encoding length and size. It's somehow explained on the "Data types" section of this document, but btfPtr is not explicitly mentioned.